src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1212,6 +1320,8 @@ pub struct QemuCmdLine<'a> {
     smp: Smp,
     machine: Machine,
     cpu: Cpu,
+    rtc: Rtc,
```

Comment:
I was wondering why this item is added here even if it is add to `devices` below.

Reply:
This is certainly a typo because `rtc` was just removed from there by #9571 :wink:

Reply:
sorry, I will fix it.